opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/entrypoint/authorization/Xs2aDenyAuthorization.java

```diff
@@ -41,7 +42,11 @@
             throw new IllegalStateException("Context not found");
         }
 
-        ctx.setRequestId(serviceContext.getRequest().getFacadeServiceable().getRequestId().toString()); // traceability
+        UUID xRequestId = serviceContext.getRequest().getFacadeServiceable().getRequestId() != null
```

Comment:
It is better to make xRequestId mandatory on API level, as it should be

Reply:
xRequestId is always notNull  in APIs